src/app/lib/printing-engine.ts

```diff
@@ -293,3 +293,47 @@ export const executePaypalPayment = async (
   )
   return response.json
 }
+
+export enum OrderStatusType {
+  ORDER_PLACED = 'placed',
+  PRODUCTION_STARTED = 'in_production',
+  ORDER_SHIPPED = 'shipped',
+  ORDER_RECEIVED = 'delivered'
```

Comment:
Please change to `received`
```suggestion
  ORDER_RECEIVED = 'received'
```